app/components/employer-table/component.js

```diff
@@ -5,8 +5,8 @@ var DATE_FORMAT = 'YYYY-MM-DD';
 
 export default Ember.Component.extend(ShowMore, {
   items: Ember.computed.alias('model.top'),
-  displayDownload: Ember.computed('exportSet', 'session.isLoggedIn', function(){
-    return this.get('session.isLoggedIn') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
+  displayDownload: Ember.computed('exportSet', 'session.isAdmin', function(){
+    return this.get('session.isAdmin') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
```

Comment:
@johnnyoshika I'd like it restricted to admins only now. It's a lot of data to give away and the amount of times BCIC logs in, I think we're ok. I can always make Bonnie an admin if need be or just send her a downloaded copy. They usually email me when they want those things anyway :).